src/utils/getQuery/getQuery.ts

```diff
@@ -0,0 +1,16 @@
+import { ListingSearchValues } from 'src/types';
+
+export const getAccommodationsListQuery = (queryParams: ListingSearchValues): string => {
```

Comment:
try using `URLSearchParams` https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams. It may help

Reply:
done